### DISCUSSION TITLE
Please give us SCOPED LABELS (grouped labels where only one can be used)

### DISCUSSION THREAD

**Discussion by alystair** [community/community#75170] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

Scoped labels are the primary reason we are still using GitLab for project management - as it trivializes issue processes/steps management that would otherwise require GitHub Actions or bots. Even [Gitea](https://docs.gitea.com/next/usage/labels#scoped-labels) has a basic implementation.

![An assortment of well-scoped labels in GitLab](https://github.com/community/community/assets/159109/25524d2c-6f2c-40c9-aa92-e05d341a966a)

GitLab scoped label uses a double-colon (::) syntax, whilst Gitea uses a slash (/).

https://docs.gitlab.com/ee/user/project/labels.html#scoped-labels

![Example of GitLab scoped labels](https://github.com/community/community/assets/159109/7f23a587-c4c5-491b-bd9c-4c2d7214cbdc)

There's a previous [general discussion](https://github.com/orgs/community/discussions/16682
) which shows some good user traction.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by itshypax** [comment]
I've already [posted about this](https://github.com/orgs/community/discussions/16682) in May 2022. Unfortunately, there was no response.
